Automattic/newspack-plugin
# feat: search credit metadata on media library

### All Submissions:






### Changes proposed in this Pull Request:



Also look in the credits metadata when searching for media in the Media library.

### How to test the changes in this Pull Request:

1. Go to Media > Library
2. Edit a couple of media items and add some information to the credits fields (Make sure those strings don't exist in the post title or content)
3. Perform a search typing part of the strings that you added to the credits fields
4. confirm that the media with those credits are returned
5. perform another search with a string present in the title of other media and confirm it still works as usual
6. perform other combination of searches and confirm they all work (a string only in the meta, a string only in post tittle, a string in both places, etc)

### Other information:
